keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -339,32 +632,43 @@ The below command should show maxUnavailable value:
 kubectl get statefulsets -o yaml | grep maxUnavailable
 ```
 
+Or refer to the new metric `rolling-update-duration`, it should exist.
+
 ###### How can someone using this feature know that it is working for their instance?
-TODO when we reach Beta
+
+With feature enabled, set the `maxUnavailable` great than 1, and pay attention to the rolling update pods at a time,
+it should equal to the `maxUnavailable`.
+Or when setting the `maxUnavailable` great than 1, the `rolling-update-duration` should decrease.
 
 ###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
 
+I think it has little relevance with SLOs, but rolling update at a very low speed which impacts the running services.
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
+None.
```

Comment:
What about `rolling-update-duration` mentioned above?

Reply:
It only works during rolling update, do you think we should count for that?

Reply:
I'll repeat my doubt about the chosen metric, I can't find it, although it seems like one that should be exposed in during beta promotion.

Reply:
Added the metric, it located at the implementation details.